db/dcrsqlite/sync.go

```diff
@@ -278,10 +330,25 @@ func (db *wiredDB) resyncDB(quit chan struct{}, blockGetter rpcutils.BlockGetter
 			return i - 1, fmt.Errorf("Unable to store stake info in database: %v", err)
 		}
 
-		// update height, the end condition for the loop
+		// Update height, the end condition for the loop
 		if _, height, err = db.client.GetBestBlock(); err != nil {
 			return i, fmt.Errorf("GetBestBlock failed: %v", err)
 		}
+
+		// If updating explore is activated, update it at intervals of 200 blocks.
+		if updateExplorer != nil && i%200 == 0 && explorer.SyncExplorerUpdateStatus() && db.updateStatusSync {
+			updateExplorer <- &blockhash
```

Comment:
Here's a spot where you could try to send the status update to the api too.

```go
select {
	case db.updateStatusChan <- summary.Height:
	default:
	}
```

Reply:
On this part I don't quite follow. On the code I can see where channel `db.updateStatusChan` is written with data but no where its data is read from the channel. What is the purpose of `db.updateStatusChan` if no data it ever read from it?

Reply:
https://github.com/decred/dcrdata/blob/acd8da6aecb0dd4821b4e70117fe8803ed8bbd73/main.go#L154